app/views/admin/tracks/index.html.haml

```diff
@@ -6,59 +6,61 @@
         Categorize events in your conference
 .row
   .col-md-12
-    %table.table.table-hover#tracks
+    %table.table.table-hover.table-striped.table-bordered.datatable#tracks
       %thead
         %th Name
-        %th Short name
         %th Description
+        %th Room
+        %th Start Date
+        %th End Date
         %th Submitter
-        %th Color
+        %th Included in Cfp
         %th State
-        %th Included in the Cfp
         %th Actions
       %tbody
         - @tracks.each do |track|
           %tr
-            %td
-              = link_to(admin_conference_program_track_path(@conference.short_title, track)) do
-                = track.name
-            %td
-              = track.short_name
+            %td{style: "padding: 15px 0px 0px 10px;"}
+              = link_to admin_conference_program_track_path(@conference.short_title, track), class: 'btn' do
+                %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
+                  = track.name
             %td
               %p
-                = truncate(track.description)
+                = markdown(truncate(track.description))
             %td
-              - if track.self_organized?
-                = link_to track.submitter.name, admin_user_path(track.submitter)
-              - else
-                N/A
+              = link_to track.room.name, admin_conference_venue_room_path(@conference.short_title, track.room.id) if track.room
             %td
-              %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
-                = track.color
+              = track.start_date.strftime('%A, %B %-d. %Y') if track.start_date
             %td
-              - if track.self_organized?
-                = track.state
-              - else
-                N/A
+              = track.end_date.strftime('%A, %B %-d. %Y') if track.end_date
             %td
+              = link_to track.submitter.name, admin_user_path(track.submitter) if track.self_organized?
```

Comment:
This potentially throws an exception

```
ActiveRecord::RecordNotFound in Admin::UsersController#show
Couldn't find Conference with 'id'=152

```
app/models/user.rb:168:in `block in get_roles'
app/models/user.rb:167:in `get_roles'
app/views/admin/users/show.html.haml:25:in `block in _app_views_admin_users_show_html_haml___579614431558334695_133750840'
app/views/admin/users/show.html.haml:18:in `each'
app/views/admin/users/show.html.haml:18:in `_app_views_admin_users_show_html_haml___579614431558334695_133750840'